componets/buttons/buttons.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
can add more meta information to give it a personalise touch.. 
like <meta name="author" content="Your Name">
<meta name="description" content="about your ui">

Reply:
I don't know more about it i will read about it and do the changes

Reply:
ok i will do it thanks